src/mumble/AudioInput.cpp

```diff
@@ -1173,8 +1174,16 @@ void AudioInput::flushCheck(const QByteArray &frame, bool terminator, std::int32
 		// accordingly once the client whispers for the next time.
 		Global::get().iPrevTarget = 0;
 	}
-	if (Global::get().s.lmLoopMode == Settings::Server) {
-		audioData.targetOrContext = Mumble::Protocol::ReservedTargetIDs::SERVER_LOOPBACK;
+
+	switch (Global::get().s.lmLoopMode) {
+		case Settings::ServerOnly:
+			audioData.targetOrContext = Mumble::Protocol::ReservedTargetIDs::SERVER_LOOPBACK_ONLY;
+			break;
+		case Settings::ServerRegular:
+			audioData.targetOrContext = Mumble::Protocol::ReservedTargetIDs::SERVER_LOOPBACK_REGULAR;
+			break;
+		default:
+			break;
```

Comment:
We should not include `default` cases when switching over enums as that will prevent compiler warnings once new enum values are added.